kolibri/plugins/learn/assets/src/views/SearchFiltersPanel/index.vue

```diff
@@ -371,6 +373,7 @@
 
   .section {
     margin-top: 40px;
+    margin-bottom: 60px;
```

Comment:
Seems like some spacing was added here? This seems like it is causing the extra space that @pcenov commented on?

Reply:
Ahh, nice catch. Just realized this class is used allover the search panel. Let me streamline this.